tests/unit/backend/utils/field-validation/attachment-validation.spec.js

```diff
@@ -34,59 +34,59 @@ describe('Attachment validation', () => {
     it('should disallow submission with no attachment if it is required', () => {
       const formField = makeField(fieldId, '1')
       const response = makeResponse(fieldId, undefined)
-      const testFunc = () => validateField(formId, formField, response)
-      expect(testFunc).toThrow()
+      const testFunc = validateField(formId, formField, response)
```

Comment:
nit: rename `testFunc` -> `validateResult` for all the changed tests. They are not functions anymore haha